src/docs/frontend/using-styled-components.mdx

```diff
@@ -0,0 +1,221 @@
+---
+title: Using Styled Components
+---
+
+To apply styles to HTML we use a css-in-js library called [Emotion](https://emotion.sh/). This allows us to define styles in the same file where the component is defined and used. Styles are scoped to avoid global selectors and having to manually specify classnames at strings. You can also leverage the theme object to reference common values and utilities.
```

Comment:
```suggestion
To apply styles to HTML, we use a CSS-in-JS library called [Emotion](https://emotion.sh/). This allows us to define styles in the same file where the component is defined and used. Styles are scoped to avoid global selectors and having to manually specify classnames as strings. You can also leverage the `theme` object to reference common values and utilities.
```